docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -271,25 +261,57 @@ spec:
 ```
 
 
-## RBAC for Sessions
+## Teleport Resources
+
+RBACs lets teams limit what resources are available to teleport users. This can be
+helpful if you don't want regular users editing SSO `auth_connector` or creating and
+editing new roles `role`.
 
-As shown in the role example above, a Teleport administrator can restrict
-access to user sessions using the following rule:
+As list of all options is below.
 
 ```yaml
-rules:
-  - resources: [session]
-    verbs: [list, read]
+  allow:
+    rules:
+    # Can create and list Teleport Roles.
+    - resources:
+      - role
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    # Can create and edit SSO connectors.
+    - resources:
+      - auth_connector
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - session
+      verbs:
+      - list
+      - read
+    # Can view trusted clusters. If update / create is set the users can create.
+    - resources:
+      - trusted_cluster
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    # Can view the audit log and session recordings.
+    - resources:
+      - event
+      verbs:
+      - list
+      - read
 ```
 
-* "list" determines if a user is allowed to see the list of past sessions.
-* "read" determines if a user is allowed to replay a session.
-
-It is possible to restrict "list" but to allow "read" (in this case a user will
-be able to replay a session using `tsh play` if they know the session ID)
```

Comment:
I think this is still true, why'd we remove it?

Reply:
I've put this back in as a comment under the RBAC resource